webapp/channels/src/components/card/card.tsx

```diff
@@ -25,7 +25,7 @@ export default class Card extends React.PureComponent<Props> {
         const childrenWithProps = Children.map(children, (child) => {
             // Checking isValidElement is the safe way and avoids a TS error too.
             if (isValidElement(child)) {
-                return cloneElement(child, {expanded});
+                return cloneElement(child, {expanded} as any);
```

Comment:
What's the type issue here?

Reply:
`cloneElement` is cloning "any" element, so `expanded` is not a prop of "any element". I found no way to make this one take the expected type from the child.

Reply:
`unknown` doesn't work? The problem with `any` is that it removes ~any~ and all type checking. It might be okay here since `cloneElement` expects it, but I try to avoid casting `as any` where possible.

Reply:
Found a way. It is not much more "type safer" as the `as any`, but looks better. :smile:

Reply:
I love generics!